src/renderers/webgl/WebGLMaterials.js

```diff
@@ -17,7 +16,7 @@ function WebGLMaterials( renderer, properties ) {
 
 	function refreshFogUniforms( uniforms, fog ) {
 
-		fog.color.getRGB( uniforms.fogColor.value, getUnlitUniformColorSpace( renderer ) );
```

Comment:
Nit: The default value (`THREE.ColorManagement.workingColorSpace`) would be the best choice. There's a mistake in the Color#getRGB documentation, to be fixed by https://github.com/mrdoob/three.js/pull/26210.

Reply:
@Oletus has this been resolved?